patch/kernel/archive/meson64-6.6/overlay/meson-g12a-radxa-zero-spi-spidev.dts

```diff
@@ -0,0 +1,38 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "radxa,zero", "amlogic,g12a";
+
+	fragment@0 {
+		target = <&spicc0>;
+		__overlay__ {
+			pinctrl-0 = <&spicc0_x_pins &spicc0_ss0_x_pins>;
+			pinctrl-names = "default";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			spidev@0 {
+				compatible = "spidev";
```

Comment:
Hey @pyavitz -- I don't this will work, upstream has disabled generic spidev devices for a while.
We carry a patch (originally from rpi, I think) that re-enables `armbian,spidev` [here](https://github.com/armbian/build/blob/main/patch/kernel/archive/meson64-6.6/drv-spi-spidev-remove-warnings.patch) - same is used on rockchip and allwinner AFAIK.

Reply:
> Hey @pyavitz -- I don't this will work, upstream has disabled generic spidev devices for a while. We carry a patch (originally from rpi, I think) that re-enables `armbian,spidev` [here](https://github.com/armbian/build/blob/main/patch/kernel/archive/meson64-6.6/drv-spi-spidev-remove-warnings.patch) - same is used on rockchip and allwinner AFAIK.

I'm not so sure about the current state of these overlays working on other units?. At least not when it comes to the current state of the DTS's. Some of the DTS files for example have these UART nodes already in them and would only need a `status = "okay";` for them to work. I'm also not sure how many of them have a `&spicc1`? I don't think the ODROID's do.

In any case, sure some of these could be used else where. My thought was just to drop in the vendor overlays and change what needed to be changed to make them work on upstream Linux and not bring any more confusion to them.

Either way I'm kool with whatever, but we would need to sort through the other patches and what not to make sure nothing conflicts.

Reply:
Yeah, but this one (`spidev` vs `armbian,spidev`) is not gonna work, regardless of the others.